public/model/layerRenderController.ts

```diff
@@ -44,9 +44,9 @@ export const prepareDataLayerSource = (
   layer: MapLayerSpecification,
   mapState: MapState,
   { data, toastNotifications }: MapServices,
-  filters: Filter[] = [],
-  timeRange?: TimeRange,
-  query?: Query
+  filtersFromDashboard: Filter[] = [],
```

Comment:
Shall we create a new type with props from dashboard?

Reply:
added